### PR TITLE
Handle missing input data and retry database queries more times

### PR DESCRIPTION
**Pull request by stdavis**

### CODE REVIEW

File: src/masquerade/providers/open_sgid.py [hunk 1]
```diff
@@ -25,8 +25,8 @@
 NAME = 'name'
 
 RETRY_WAIT_MIN = 0.5
-RETRY_WAIT_MAX = 5
-RETRY_ATTEMPTS = 3
+RETRY_WAIT_MAX = 3
```
Comment (steveoh) [inline src/masquerade/providers/open_sgid.py:28]: **question**: is this seconds?

**Review by steveoh — APPROVED** [review]

File: setup.py [hunk 2]
```diff
@@ -59,7 +59,7 @@
             'pytest-watch==4.2.*',
             'pytest==7.3.*',
             'requests-mock==1.10.*',
-            'yapf==0.33.*',
+            'yapf==0.40.*',
```
[the remark this answers is not in the document]
Comment (stdavis) [reply setup.py:62]: Oh yes. Thanks for the reminder. I will make the switch in this PR.

**Comment by stdavis** [comment]
@steveoh This is ready for another look when you have a minute.

**Review by steveoh — APPROVED** [review]

File: .vscode/settings.json [hunk 3]
```diff
@@ -3,12 +3,7 @@
   "python.linting.pylintEnabled": true,
   "python.linting.flake8Enabled": false,
   "python.linting.pycodestyleEnabled": false,
-  "python.formatting.provider": "yapf",
-  "python.formatting.yapfPath": "yapf",
-  "python.formatting.yapfArgs": [
-    "--style",
-    "${workspaceRoot}/setup.cfg"
```
Comment (steveoh) [inline .vscode/settings.json:10]: can setup.cfg be removed now?

**Review by steveoh — COMMENTED** [review]
I kept pylint with black. It seemed like an eslint + prettier combo. Looks like you just merged them into the toml. I see. nice.

**Pull request merged**